kafka/admin/client.py

```diff
@@ -460,6 +460,28 @@ def create_topics(self, new_topics, timeout_ms=None, validate_only=False):
         # TODO raise exceptions if errors
         return self._send_request_to_controller(request)
 
+    def list_partition_reassignments(self, topics_partitions, timeout_ms=None):
```

Comment:
Given the complexity, I might suggest splitting this into a PR that adds support for KIP-482 at the protocol level and another PR that adds the new admin APIs.

Reply:
removed, will add in another PR